### ISSUE TITLE
Download?

### ISSUE THREAD

**Issue by kazimpal87** [JeremyXSC/GPR#1]
Hi, is the dataset available for download anywhere? Thanks

**Comment by JeremyXSC** [comment]
So far, this work is still in progress, in the future, we will consider to publish this dataset with the consent of the partnet when all the works are finished. More details about this dataset are also available at the project page: https://JeremyXSC.github.io/GPR/.
Thank you!

**Comment by Glasses0** [comment]
Hi,the dataset has not been released yet？Is it available for dowload？Thanks

**Comment by JeremyXSC** [comment]
Sorry for the late reply!
Thanks for your interest in our work. For your questions, we will try our best to release the whole dataset as soon as possible. Besides, some examples of the fine-grained version of GPR ( named as "FineGPR") is also available at https://github.com/JeremyXSC/FineGPR. Thanks for your understanding.

**Comment by IceIce1ce** [comment]
When the dataset be released?

**Comment by JeremyXSC** [comment]
Thank you for your continuous interest in our work. The full version of our GPR dataset (named as "FineGPR") is available at https://github.com/JeremyXSC/FineGPR. I hope it will be helpful for your research!

**Comment by IceIce1ce** [comment]
Hi author,

Thanks for your information. Also, I found that only the one drive link is provided for the small subset. Does the full version have a google drive or one drive link?

**Comment by JeremyXSC** [comment]
Yes, the full version of FineGPR dataset can be downloaded from [Baidu Yun Drive](https://pan.baidu.com/s/18cQXtGqJKLjU7HpX5o9gWQ) password: jpr9

**Comment by IceIce1ce** [comment]
Currently, I do not have a Baidu account. Therefore, I cannot download the full version of the dataset from this Baidu Yun drive. Can you update a one drive download link for the full version?

**Comment by JeremyXSC** [comment]
I am very sorry for that. You can either register a Baidu account yourself or ask a colleague to help you with the dataset download. As for updates, there might be a waiting period. Thanks!

**Comment by IceIce1ce** [comment]
Thanks for your information. Anyway, thanks for your support.

**Comment by JeremyXSC** [comment]
My pleasure~
